packages/honeycomb-opentelemetry-web/CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # honeycomb-opentelemetry-web changelog
 
+## v0.18.0 [beta] - 2025-05-20
+
+### 💥 Breaking Changes
+- Upgrade to web-vitals 5.0.0 (chore: Upgrade to web-vitals 5.0.0 #534) removes support for FID since it has been removed from the web-vitals@5.0.0 for the full set of upstream changes see changelog here
```

Comment:
I don't think the link copied
```suggestion
- **chore: Upgrade to web-vitals 5.0.0 (#534)** removes support for FID since it has been removed from the `web-vitals@5.0.0` for the full set of upstream changes see changelog [here](https://github.com/GoogleChrome/web-vitals/blob/main/CHANGELOG.md#v500-2025-05-07)
```